src/frontend/src/components/OeciLogin/index.tsx

```diff
@@ -1,22 +1,21 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { AppState } from '../../redux/store';
-import { logIn } from '../../redux/system/actions';
+import { OECILogIn } from '../../redux/system/actions';
```

Comment:
If you grep (case-sensitive) for Oeci and OECI in our code:

```
mkw@M4600:~/Documents/repos/re-pdx/recordexpungPDX/src/frontend$ gpnifr Oeci
./src/components/OeciLogin/index.tsx:class OeciLogin extends React.Component<Props, State> {
./src/components/OeciLogin/index.tsx:            id="OeciLoginForm"
./src/components/OeciLogin/index.tsx:)(OeciLogin);
./src/components/AppRouter/index.tsx:import OeciLogin from '../OeciLogin';
./src/components/AppRouter/index.tsx:          <AuthenticatedRoute path="/oeci" component={OeciLogin} />

mkw@M4600:~/Documents/repos/re-pdx/recordexpungPDX/src/frontend$ gpnifr OECI
./src/components/PublicRoute/index.tsx:    // if authenticated it redirects to the OECI login
./src/components/OeciLogin/index.tsx:            aria-label="OECI Login Form"
./src/components/OeciLogin/index.tsx:                Log in to OECI to search and analyse criminal records for
```

Note that "OECI" is used only in comments and user-facing copy. 

We should follow the existing capitalization convention and use Oeci instead of OECI, acronyms aren't exceptions to the rule.

https://github.com/airbnb/javascript#naming-conventions

Reply:
I will rename this!

I think I was just confusing myself thinking that since it was an acronym the `ECI` would need to be capitalized but `oECILogIn` looked pretty strange.

I think the proper naming convention for a function would just be standard camelCase `oeciLogIn`